ui/open-api-ui/src/test/java/io/smallrye/openapi/ui/IndexCreatorTest.java

```diff
@@ -240,4 +240,17 @@ void testCreateWithSyntaxHighlightObject() throws IOException {
 
         assertTrue(s.contains("syntaxHighlight: { activated: true, theme: \"monokai\" },"));
     }
+
+    @Test
+    void testCreateWithTryItOutEnabledBoolean() throws IOException {
```

Comment:
The default for `tryItOut` is enabled (see https://swagger.io/docs/open-source-tools/swagger-ui/usage/configuration/) so I would rather test `false` or `disabled`.

Reply:
The test only test that the value is set, so maybe this is actually fine.